modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -199,12 +214,14 @@ inline void preprocess(const cv::Mat& src,
                        const cv::gimpl::onnx::TensorInfo& ti,
                              cv::Mat& dst) {
     GAPI_Assert(src.depth() == CV_32F || src.depth() == CV_8U);
-
+    const auto type = toCV(ti.type) != -1
```

Comment:
Why does it became `type` ?

Reply:
It was type and now it is type.
toCV(ti.type) for if()
and
const auto ddepth = toCV(ti.type) for preprocessing - the same value.